docs/user-guide/src/version_support.md

```diff
@@ -18,10 +20,10 @@ Currently, in Metal³ organization only CAPM3 and IPAM follow CAPI release cycle
 
 - CAPM3
    - v1beta1
-      - v1.4.0, v1.3.2, v1.3.1, v1.3.0, v1.2.2, v1.2.1, v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0
+      - v1.5.0, v1.4.3, v1.4.2, v1.4.1, v1.4.0, v1.3.4, v1.3.3, v1.3.2, v1.3.1, v1.3.0, v1.2.2, v1.2.1, v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0
```

Comment:
It feels waste of time to specify up to patch level one by one as we're guaranteeing not to make breaking changes in patches. Could we just have 'v1.5.x`, `1.4.x` etc?